spec

```diff
@@ -508,17 +508,15 @@ beginseg
     include "build/data/code_800FFA50.bss.o"
     include "build/asm/code_80100040.o"
     include "build/asm/code_801001E0.o"
-    include "build/asm/code_80100290.o"
-    include "build/data/code_80100290.data.o"
-    include "build/data/code_80100290.rodata.o"
-    include "build/data/code_80100290.bss.o"
```

Comment:
I may just be missing them but I don't see those files removed in the PR, which they should be if they are no longer used in the spec.